miner/src/miner.rs

```diff
@@ -228,6 +228,11 @@ impl MinerService for Miner {
 		queue.find(hash)
 	}
 
+	fn pending_transactions(&self) -> Vec<SignedTransaction> {
```

Comment:
I think I would call it `top_transactions` to avoid confusion with block-pending transactions (but maybe it's just my wrong understanding of this wording).